### PR TITLE
Remove pipeline config api v3 and v4

### PR DESCRIPTION
**Pull request by ketan**

### CODE REVIEW

File: server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/internal/material_test_controller.rb [hunk 1]
```diff
@@ -25,10 +25,10 @@ class MaterialTestController < ::ApiV1::BaseController
         before_action :check_admin_user_or_group_admin_user_and_403
 
         def test
-          material_config = ApiV3::Admin::Pipelines::Materials::MaterialRepresenter.new(ApiV3::Admin::Pipelines::Materials::MaterialRepresenter.get_material_type(params[:type]).new).from_hash(params)
+          material_config = ApiV4::Admin::Pipelines::Materials::MaterialRepresenter.new(ApiV4::Admin::Pipelines::Materials::MaterialRepresenter.get_material_type(params[:type]).new).from_hash(params)
```
Comment (varshavaradarajan) [inline server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/internal/material_test_controller.rb]: The material_test_controller was for the check connection button on the quick edit page, right? Why didn't you want this to be ApiV5::Admin::Pipelines::Materials::MaterialRepresenter?

Comment (varshavaradarajan) [reply server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/internal/material_test_controller.rb]: You can get rid of the materials directory in api_v4 then

Comment (ketan) [reply server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/internal/material_test_controller.rb]: It's also used by `ConfigRepoOriginRepresenter`, want me to change that usage too?

Comment (varshavaradarajan) [reply server/webapp/WEB-INF/rails.new/app/controllers/api_v1/admin/internal/material_test_controller.rb]: Yes please.

File: server/webapp/WEB-INF/rails.new/app/presenters/api_v4/shared/config_origin/config_repo_origin_representer.rb [hunk 2]
```diff
@@ -28,7 +28,7 @@ def type
         end
 
         def repo
-          material = ApiV4::Admin::Pipelines::Materials::MaterialRepresenter.new(config_repo.getMaterial).to_hash(url_builder: self)
+          material = ApiV5::Admin::Pipelines::Materials::MaterialRepresenter.new(config_repo.getMaterial).to_hash(url_builder: self)
```
Comment (varshavaradarajan) [inline server/webapp/WEB-INF/rails.new/app/presenters/api_v4/shared/config_origin/config_repo_origin_representer.rb]: Isn't this representer used by the pipeline config api v4 alone? config repo representation is only present for environments and pipelines. If there is no other representer using api_v4 config repo origin representer, you can get rid of this as well?

**Pull request merged**